### DISCUSSION TITLE
How to Store Bytes-Type Data Using prisma-client-py

### DISCUSSION THREAD

**Discussion by ghost** [RobertCraigie/prisma-client-py#1032] [Q&A] [answered]
## Hi,

I am working on a project that involves storing binary data (Bytes) in a PostgreSQL database using prisma-client-py version 0.15.0. I encountered an issue regarding the proper way to store binary data in the database. Here is a detailed description of the problem and the solution I have attempted:
My Prisma schema file

![schema](https://github.com/user-attachments/assets/156a2446-bf0d-4535-989d-2085062a6c41)
I attempted to upload a file and store its content in the Score model using the following Python script:

![scores](https://github.com/user-attachments/assets/d4f8bf51-389e-40de-87f8-bd29d0f266a8)
I expected the script to connect to the PostgreSQL database and create a new Score record with the uploaded file's content stored correctly.
How can I resolve the type error warning given by VSCode?
Thank you for your help and insights.

**Comment by RobertCraigie** [comment, marked as answer]
Here are some examples: https://prisma-client-py.readthedocs.io/en/stable/reference/operations/#bytes-fields, let me know if that helps!

**Reply by ghost** [reply]
thank you 🙏
